### PR TITLE
92 - Updated missing token icons

### PR DESCRIPTION
**Pull request by mukulshakya**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

About # (link your issue here)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/constants/fuseTokenMapping.ts [hunk 1]
```diff
@@ -46,6 +46,16 @@ export const getFuseTokenLogoURL = (address: string) => {
...
+    // G$ // TODO - reads from fuse staging, handling for production?
+    if (address?.toLowerCase() === '0x100b8fd10ff8DC43fda45E636B4BB1eE6088270a'.toLowerCase()) {
+        uri = 'https://raw.githubusercontent.com/GoodDollar/GoodProtocolUI/master/src/assets/images/tokens/gd-logo.png'
+    }
+
+    // Good // TODO - reads from fuse staging, handling for production?
+    if (address?.toLowerCase() === '0xf4467Ad0c9D7D0446B379fE6217111a4690066D1'.toLowerCase()) {
+        uri = 'https://raw.githubusercontent.com/GoodDollar/GoodProtocolUI/master/src/assets/images/tokens/good-logo.png'
+    }
+
```
Comment (sirpy) [inline src/constants/fuseTokenMapping.ts:58]: on fuse G$ and GOOD have 3 different addresses for the 3 envs we have dev,staging,production
in deployment.json dev=fuse staging=staging production=production
on ropsten G$ GOOD and G$X has two addresses  fuse-mainnet and staging-mainnet
kovan G$ GOOD G$X kovan-mainnet
mainnet G$ GOOD G$X production-mainnet
@mukulshakya @adaptivecodeio

**Review by sirpy — CHANGES_REQUESTED** [review]

File: src/constants/fuseTokenMapping.ts (hunk 1 above)
Comment (mukulshakya) [reply src/constants/fuseTokenMapping.ts]: Hi @sirpy 

Thank you for confirming the addresses. I have updated all of them please check

File: src/constants/fuseTokenMapping.ts [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
+    const G$Addresses = [
+        '0xeE724540706296ebad65aeA2515Efe0949F97Ae6', // dev
+        '0x100b8fd10ff8DC43fda45E636B4BB1eE6088270a' // staging
+    ].map(address => address.toLowerCase())
+    const GoodAddresses = [
+        '0x0Fce4a964F2b69a6cD82c3FB40C101863091A5a7', // dev
+        '0xf4467Ad0c9D7D0446B379fE6217111a4690066D1' // staging
+    ].map(address => address.toLowerCase())
 
     if (address?.toLowerCase() === '0x495d133B938596C9984d462F007B676bDc57eCEC'.toLowerCase()) {
```
Comment (sirpy) [inline src/constants/fuseTokenMapping.ts:14]: this is fuse production
for ethereum production simply search etherscan for GoodDollar

**Review by sirpy — CHANGES_REQUESTED** [review]

File: src/constants/fuseTokenMapping.ts (hunk 2 above)
Comment (mukulshakya) [reply src/constants/fuseTokenMapping.ts:14]: @sirpy i didn't follow here can you elaborate a little?

Comment (sirpy) [reply src/constants/fuseTokenMapping.ts:14]: 0x495d133B938596C9984d462F007B676bDc57eCEC this is fuse production G$ address
you should have an array for G$X GOOD and G$ with all envs addresses and not in a separate if like in line 14.

you wrote in TODO that you are missing production data
you can get all production addresses from the latest GoodDollar/GoodProtocol/release/deployment.json

Comment (mukulshakya) [reply src/constants/fuseTokenMapping.ts:14]: Thank you i will update it

Comment (sirpy) [reply src/constants/fuseTokenMapping.ts:14]: @mukulshakya well?

Comment (mukulshakya) [reply src/constants/fuseTokenMapping.ts]: @sirpy Updated

**Comment by sirpy** [comment]
@imanshu03 i dont see where did you put the production data and dont understand from the logic how is it going to display the tokens images on mainnet
also please attach screenshots of it working on mainnet/ropsten/fuse/kovan

**Comment by imanshu03** [comment]
@sirpy I'm not the dev for this ticket. @mukulshakya has worked on this, He will be able to answer your above query.

**Comment by mukulshakya** [comment]
> @imanshu03 i dont see where did you put the production data and dont understand from the logic how is it going to display the tokens images on mainnet also please attach screenshots of it working on mainnet/ropsten/fuse/kovan

@sirpy i think i just need to update the version in package.json to the latest release from **GoodDollar/GoodProtocol**

Also Ethereum mainnet was disable since the contact addresses was not present before. So do i need to enable it again?

**Comment by sirpy** [comment]
yes update to the latest, ethereum should work when network env is set to production

**Comment by mukulshakya** [comment]
> yes update to the latest, ethereum should work when network env is set to production

@sirpy updated the latest pkg

**Review by sirpy — COMMENTED** [review]
@mukulshakya have you verified this now worrks  correctly on all networks including mainnet?
if so you can squash merge it

**Comment by mukulshakya** [comment]
@sirpy wanted to confirm 1 more thing.

Case for MAINNET is empty, should i leave it as it is if no the what will be the correct deployment name for mainnet?
image

**Comment by sirpy** [comment]
@mukulshakya mainnet contracts is configured just fine, i've asked you about the icons if you  verified it works on Ethereum correctly

**Comment by mukulshakya** [comment]
> @mukulshakya mainnet contracts is configured just fine, i've asked you about the icons if you verified it works on Ethereum correctly

@sirpy i have verified and the icons are showing correctly

**Pull request merged**